src/components/avatar/UnknownUserAvatar.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import AccessibleSVG, { SVGProps } from '../accessible-svg/AccessibleSVG';
+import * as vars from '../../styles/variables';
 
 const UnknownUserAvatar = ({ className = '', height = 28, title, width = 28 }: SVGProps) => (
```

Comment:
Should this component wrap the icon comp instead of duplicating it?

Reply:
These are pretty much identical, but they each use different versions of `AccessibleSVG` components with incompatible props. The one here references the `AccessibleSVG` in `src/components/accessible-svg/AccessibleSVG.tsx` which is marginally different from the accessible svg in `src/icons/accessible-svg/AccessibleSVG.tsx` in the icon comp. We should probably standardize the props between these two components as they functionally serve the same purpose, but I wouldn't think that would be in scope for this update.

Reply:
Can you create a ticket to track that clean up work?

Reply:
https://github.com/box/box-ui-elements/issues/2840 created

Reply:
Thanks - but move that to internal Jira. We don't track tasks in this repo.